core/datastore/build.gradle.dcl

```diff
@@ -0,0 +1,29 @@
+androidLibrary {
+    namespace = "com.google.samples.apps.nowinandroid.core.datastore"
+
+    // TODO: We really want to model a list of consumer proguard files here, but can't yet
+    consumerProguardFile = "consumer-proguard-rules.pro"
+
+    dependencies {
+        api("androidx.datastore:datastore:1.0.0")
+        api(project(":core:datastore-proto"))
+        api(project(":core:model"))
+
+        implementation(project(":core:common"))
+    }
+
+    hilt {}
+
+    testing {
+        dependencies {
+            implementation(project(":core:datastore-test"))
+            implementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:1.8.0")
+        }
+
+        testOptions {
+            returnDefaultValues = true
```

Comment:
💅 It would be nicer if the Kotlin naming convention for boolean properties was preserved. People already have to deal with it when migrating from Groovy to Kotlin and this might add more friction.

Reply:
I do think the current way is more "declarative" looking, but I understand your point.

@eskatos, @h0tk3y - passing along some feedback.

Reply:
+1 to Rodrigo's comment!

@h0tk3y, could you please take care of creating an issue about making sure our schema extraction complies with Kotlin naming convention for boolean properties?